Include/httpClient/httpClient.h

```diff
@@ -702,3 +705,17 @@ hc_websocket_handle_t HCWebSocketDuplicateHandle(
 STDAPI HCWebSocketCloseHandle(
     _In_ hc_websocket_handle_t websocket
     ) HC_NOEXCEPT;
+
+/// <summary>
+/// Used to wrap the JavaVM and ApplicationContext on Android devices.
+/// </summary>
+#if HC_PLATFORM == HC_PLATFORM_ANDROID
+typedef struct _HC_PLATFORM_INIT_ARGS {
```

Comment:
>_HC_PLATFORM_INIT_ARGS [](start = 15, length = 22)

Are we using this anywhere?

Reply:
nevermind

---
In reply to: [206337845](https://github.com/Microsoft/libHttpClient/pull/217#discussion_r206337845) [](ancestors = 206337845)